README.md

```diff
@@ -65,6 +65,19 @@ If you want to set host to a specific IP address for example `172.17.12.1` run `
 - CSS developers should ensure that animation is well documented and communicated to the respective React developer.
 - Once the component/enhancement is complete it should receive sign off from a visual designer who can then update the master sketch file with any changes.
 
+## Experimental components
+
+When creating a brand new component, it should be released as experimental in order to get feedback.  This results in the css not being bundled into the full distribution (`patternfly.css`), but the component level css is still included in the npm package for usage.
+
```

Comment:
Should we provide specific instructions here on how to add/name the style sheet associated with that experimental component? What happens if the experimental component includes updating styles from another component/or global styles?

Reply:
Let's hold off on that until we can meet to discuss how we are going to use this.  I would like to get this into use and talk to you guys about the possibilities.  There should be no updating or usage of other components or interaction of styles in the first pass of using these - these are meant to be brand new components, not modifications of existing nor changes to anything global.